README.md

```diff
@@ -102,3 +110,5 @@ client.start();
     <!-- ... -->
 </body>
 ```
+### IE11 support
+Our target version of javascript is ES2015.  This means that you will be required to transpile this library if you wish to support IE11.  In addition the necessary polyfills will need to be loaded by both the host application and the client frame.
```

Comment:
+1 for this.  I think not transpiling or including polyfills allows for maximum consumption avenues.

I think it would ultimately be cool to also compile our ts files, but output the assests as js es modules.  That allows for direct consumption of the modules by shops fully embracing es modules and a tooled build.

Internally, I think we could consume this library and build a UMD es5 version with transpiled code and polyfills for our supported browsers that could be used in require or CDN model if desired.